gunicorn/arbiter.py

```diff
@@ -51,6 +51,8 @@ class Arbiter(object):
         if name[:3] == "SIG" and name[3] != "_"
     )
 
+    last_logged_worker_count = None
```

Comment:
The instance variable is created once it's assigned to for the first time. I'd be fine moving it into `__init__`.